themes/src/main/resources/theme/base/login/register.ftl

```diff
@@ -112,17 +112,10 @@
                                class="${properties.kcLabelClass!}">${msg("passwordConfirm")}</label>
                     </div>
                     <div class="${properties.kcInputWrapperClass!}">
-                        <div class="${properties.kcInputGroup!}">
                             <input type="password" id="password-confirm" class="${properties.kcInputClass!}"
                                    name="password-confirm"
                                    aria-invalid="<#if messagesPerField.existsError('password-confirm')>true</#if>"
                             />
-                            <button class="pf-c-button pf-m-control" type="button" aria-label="${msg('showPassword')}"
```

Comment:
Same for this password reveal button, I don't think this should have been removed.